docs/docs/secrets-managers.mdx

```diff
@@ -0,0 +1,152 @@
+---
+id: secrets-managers
+sidebar_label: Secrets Managers
+title: Secrets Managers
+description: Safeguard credentials your service uses to authenticate to external resources.
+abstract: You can store your assistant's secrets in an external credentials manager.
+    Rasa Pro currently supports credentials manager for the Tracker Store
+---
+
+<!-- this file is version specific, do not use `@site/...` syntax -->
+import RasaProLabel from '@theme/RasaProLabel';
+import RasaProBanner from '@theme/RasaProBanner';
+
+<RasaProLabel />
+<RasaProBanner/>
+
+
+Rasa Pro supports the following secrets managers:
+* [HashiCorp Vault](https://www.hashicorp.com/products/vault)
+
+
+Currently, Rasa Pro supports safeguarding credentials for the following services:
+- [Tracker Stores](./tracker-stores.mdx)
+
+
+## HashiCorp Vault Secrets Manager
+
+Use Vault Secrets Manager to store credentials used to authenticate access to external services.
+The credentials are stored in a Vault instance and can be encrypted at rest.
+To store credentials in a Vault instance, you can read the official Vault docs
+[Storing secrets in Vault](https://developer.hashicorp.com/vault/tutorials/getting-started/getting-started-first-secret).
+
+You can also encrypt credentials at rest.
+you can use the [Vault Transit Engine](https://www.vaultproject.io/docs/secrets/transit).
```

Comment:
```suggestion
You can also encrypt credentials at rest.
You can use the [Vault Transit Engine](https://www.vaultproject.io/docs/secrets/transit).
```
Not sure if this should be two separate sentences, or one. If one then we should rephrase it somehow.

Reply:
@Balowen 
I updated this to 
```
You can also encrypt credentials at rest with [Vault Transit Engine](https://www.vaultproject.io/docs/secrets/transit).
```
WDYT?

Reply:
LGTM